pkg/apis/node.harvesterhci.io/v1beta1/nodeconfig.go

```diff
@@ -0,0 +1,66 @@
+package v1beta1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ConfigsApplied     = "Applied"
+	ConfigsWaitApplied = "WaitApplied"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName=nodeconfigs,scope=Namespaced
+// +kubebuilder:printcolumn:name="Scope",type=string,JSONPath=`.spec.nodeName`
+// +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.allConfigStatus`
+
```

Comment:
`status` is not a subresource. Is there a specific reason for this.

Reply:
I turn it into a subresource. Thanks!